Google.PowerShell/BigQuery/GcbqDataset.cs

```diff
@@ -98,33 +99,32 @@ protected override void ProcessRecord()
                     DatasetsResource.ListRequest lRequest = new DatasetsResource.ListRequest(Service, Project);
                     lRequest.All = All;
                     lRequest.Filter = Filter;
-                    var lResponse = lRequest.Execute();
-                    if (lResponse != null)
+                    try
                     {
+                        var lResponse = lRequest.Execute();
                         WriteObject(lResponse, true);
                     }
-                    else
+                    catch
```

Comment:
This naked catch concerns me. You can't know that it was a 400. What if it was a 500? What if GCP is down? I think the old if else is better in this case.